### PR TITLE
Upload live

### PR DESCRIPTION
**Pull request by Vesihiisi**
Uploads go live instead of to the Wikidata sandbox. Includes creating new items when necessary.

Task: https://phabricator.wikimedia.org/T159328

### CODE REVIEW

File: importer/Uploader.py [hunk 1]
```diff
@@ -246,17 +245,18 @@ def upload(self):
     def set_wd_item(self):
         item_exists = True if self.data["wd-item"] is not None else False
         if item_exists:
-            # item_q = self.data["wd-item"]
-            self.wd_item = self.wdstuff.QtoItemPage(self.TEST_ITEM)
-            print(self.wd_item)
+            item_q = self.data["wd-item"]
```
Comment (lokal-profil) [inline importer/Uploader.py]: I would probably keep the test item options. Maybe something like:
```
if self.test:
    self.wd_item = self.wdstuff.QtoItemPage(self.TEST_ITEM)
elif item_exists:
    item_q = self.data["wd-item"]
    self.wd_item = self.wdstuff.QtoItemPage(item_q)
else:
    self.wd_item = self.create_new_item(self.log)
```
That way you can toggle testing on/off (which is useful for later datasets) without having to update deep parts of the code. You could even set it as a command line argument.

Comment (lokal-profil) [inline importer/Uploader.py]: Unless you now use a different testing strategy, of course.

Comment (Vesihiisi) [inline importer/Uploader.py]: I have addressed this now; just using the --upload argument on its own will use the Sandbox mode, in order to go `live` you actually have to add that as an argument. That way you have to explicitly request the live mode, which prevents triggering it by accident. I have also updated the README.

File: importer/Uploader.py [hunk 2]
```diff
@@ -243,19 +243,33 @@ def upload(self):
...
+        """
+        Determine WD item to manipulate.
+
+        In live mode, if data object has associated WD item,
+        edit it. Otherwise, create a new WD item.
+        In sandbox mode, all edits are done on the WD Sandbox item.
+        """
+        if self.live:
+            item_exists = True if self.data["wd-item"] is not None else False
+            if item_exists:
```
Comment (lokal-profil) [inline importer/Uploader.py]: Any reason to not just use `if self.data["wd-item"] is not None` directly?

Are there situations when `if self.data["wd-item"]` is false but `self.data["wd-item"] is None` is also false?

File: importer/Uploader.py (hunk 1 above)
Comment (lokal-profil) [inline importer/Uploader.py]: That seems like a good solution to me =)

File: importer/Uploader.py (hunk 2 above)
Comment (Vesihiisi) [inline importer/Uploader.py]: The official reason is probably "I wasn't sure what would happen if I left that out and also I wanted to be explicit to myself"

Comment (Vesihiisi) [inline importer/Uploader.py]: Ok, the more I look at this the less sense it makes. I guess I thought a shorter "if" row would look better.

Comment (Vesihiisi) [inline importer/Uploader.py]: I have simplified this part.

**Pull request merged**